src/Meta/Meta_Definer.php

```diff
@@ -17,16 +18,15 @@ public function define(): array {
 				DI\get( Alert_Settings_Meta::class ),
 				DI\get( Alert_Meta::class ),
 			] ),
-
 			Alert_Settings_Meta::class  => DI\autowire()
 				->constructorParameter( 'object_types', static fn( ContainerInterface $c ) => [
 					'settings_pages' => [ $c->get( Alert_Settings::class )->get_slug() ],
 				] ),
-
 			Alert_Meta::class           => DI\autowire()
 				->constructorParameter( 'object_types', static fn() => [
 					'post_types' => [ Alert::NAME ],
-				] ),
+				] )
+				->constructorParameter( 'color_options', DI\get( Alert_Color_Options::class ) ),
```

Comment:
As discussed on slack, this isn't needed now that this works properly and can be removed: https://github.com/moderntribe/tribe-alerts/pull/8/files#diff-d6dc23067a7f08b3a796fc2347b93aed4afc503f1e5841686478d67f431b25b7R19

Reply:
Removed.